samples/sensor/ams_iAQcore/sample.yaml

```diff
@@ -2,7 +2,7 @@ sample:
   description: Demonstration of the AMS iAQ-core Digital VOC Sensor driver
   name: iAQcore sample
 tests:
-  sample.iAQcore:
+  sample.sensor.iAQcore:
```

Comment:
My suggestion is to follow the same style in all case names to print their names with lower case letters like `sample.sensor.iaqcore:`